cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -251,6 +251,9 @@ func init() {
 	MigrateApplyCmd.Flags().SortFlags = false
 	cobra.CheckErr(MigrateApplyCmd.MarkFlagRequired(migrateFlagURL))
 	MigrateApplyCmd.MarkFlagsMutuallyExclusive(migrateApplyFromVersion, migrateApplyBaselineVersion)
+	cobra.CheckErr(MigrateApplyCmd.Flags().MarkHidden(migrateFlagDirFormat))
```

Comment:
Why it's hidden?

Reply:
Because we only apply our own format. Other formats are applied by the respective tool, aren't they?